Web/Scotty/Internal/Types.hs

```diff
@@ -60,10 +71,11 @@ data ScottyState e m =
     ScottyState { middlewares :: [Wai.Middleware]
                 , routes :: [Middleware m]
                 , handler :: ErrorHandler e m
+                , routesOptions :: RouteOptions
```

Comment:
why is this field name plural where the type name is singular?

Reply:
> why is this field name plural where the type name is singular?

No reason, I fixed it to be singular